generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs

```diff
@@ -5,6 +5,7 @@
 * Make sure not to add !default to values here
 */
 
+<%_ if (clientTheme === 'none') { _%>
```

Comment:
Can you use `clientThemNone` derived variable starting from now?

Reply:
Ok, forgot about this

Reply:
I think that many members will soon hate me.

Reply:
:-D

Reply:
😅